### PR TITLE
Use SSH credentials in Jenkins

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Comment by evgeni** [comment]
Should the steps have `sshagent(credentials: ['…'])` as per (docs)[https://jenkins.io/doc/pipeline/steps/ssh-agent/]?

**Comment by evgeni** [comment]
And the new setup broke the validation we usee vis travis :(

**Comment by ekohl** [comment]
I got this by generating it in the UI and confirmed locally it worked to use it without `credentials:`

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/katello/3.10.groovy [hunk 1]
```diff
@@ -14,7 +14,7 @@ pipeline {
 
             steps {
 
-                sh "ssh -o 'BatchMode yes' root@koji.katello.org katello-mash-split-3.10.py"
+                mash("katello-mash-3.10-split.py")
```
[the remark this answers is not in the document]
Comment (ekohl) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/katello/3.10.groovy]: No, probably a side effect of a long day.

**Pull request merged**